### PR TITLE
Added GameSlice subclass for first level <feature

### PR DESCRIPTION
**Pull request by levilime**
Now the difference between being a first level and a level in sequence is
clear. It is by design that now a level can't be a first level and
a sequenced level at once.

### CODE REVIEW

**Comment by NilsHullegien** [comment]
I still get spawn killed when i start the game. 
Is this intended or is this a bug?

<review

File: core/src/main/java/com/group5/core/controllers/GameSliceSequenced.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 /**
  * GameSlice is a 2Dimensional rectangle in which objects are placed.
  */
-public abstract class GameSlice {
+public abstract class GameSliceSequenced {
```
Comment (mwolting) [inline core/src/main/java/com/group5/core/controllers/GameSliceSequenced.java]: I'd say SequencedGameSlice would be more in line with standard Java class naming conventions

File: core/src/main/java/com/group5/core/controllers/GameSliceFirst.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.group5.core.controllers;
+
+import com.badlogic.gdx.math.Vector2;
+import com.group5.core.world.WorldObject;
+import java.util.LinkedList;
+
+/**
+ * GameSlice is a 2Dimensional rectangle in which objects are placed.
+ */
+public abstract class GameSliceFirst extends GameSliceSequenced {
```
Comment (mwolting) [inline core/src/main/java/com/group5/core/controllers/GameSliceFirst.java]: FirstGameSlice?

**Comment by mwolting** [comment]
Code seems good, but some of the class names could be improved on in my opinion.

**Comment by levilime** [comment]
@NilsHullegien  @mwolting  fixed both your concerns, please merge

**Comment by NilsHullegien** [comment]
Looks good now!

**Pull request merged**